src/pages/en/guides/server-side-rendering.mdx

```diff
@@ -99,7 +99,23 @@ You can also add an adapter manually by installing the package and updating `ast
 
 ## Features
 
-Astro will remain a static-site generator by default. But once you enable a server-side rendering adapter, **every route in your pages directory becomes a server-rendered route** and a few new features become available to you.
+Astro will remain a static-site generator by default. But once you enable a server-side rendering adapter, **every route in your pages directory defaults to a server-rendered route** and a few new features become available to you.
+
+### Prerendering
+
+<Since v="2.0.0-beta.0" />
+
+Any file in the `pages/` directory can opt-in to prendering, or standard static build-time behavior, by including the following line.
```

Comment:
```suggestion
Any `.astro` file within the `pages/` directory can opt-in to prendering, or standard static build-time behavior, by including the following line.
```
Or MDX? But, we probably should specify explicitly since `.md` files are (currently!) common in `src/pages/`